src/theme/colors.ts

```diff
@@ -6,6 +6,8 @@ const color = {
   lightRed: '#F5E4E4',
   darkRed: '#6f3638',
   mediumRed: '#E6BCBC',
+  ultraLightGray: '#EFF3F4',
+  sutileGray: '#F5F5F5',
   lightGray: '#C3CDCF',
```

Comment:
a thought: I'd be nice to standardize these.

Really we don't need so many, but the design / theme system could use work